### PR TITLE
ZAYIN updates

### PR DESCRIPTION
**Pull request by Coxswain-Navigator**
## About The Pull Request
Adds new soda can sprites, shout out to /vg/station for the sprites.

So we discussed updates to ZAYIN abnormalities a while ago and most of the changes have yet to have been implemented. This PR implements those changes.

You are Bald:
- Provides user with a temporary healing that scales with the amount of bald people nearby.
- Add visual/audio effects to the BALDING process. Both for the one working and
  those affected by "mass attack".
- Fixes an oversight where bald dealt white instead of black damage.
- Reflects new changes on Distorted Form's "You're bald" form.

Bottle of Tears:
- Cake regenerates one use every meltdown.
- The maximum amount of cake available to consume has been lowered from 4 to 2.
- Consuming the last piece of cake only awards a special E.G.O. the first time.

Hammer of Light:
- On good work result selects a small portion of players with low health/sanity
  (below 50%) and applies a fast over-time healing effect, which heals
  5% hp and sp per tick, for a total of 20-30%.

Vending Machine and Oceanic Waves
- Dispenses a much larger variety of drinks with varying effects.
- No more cocaine. At least, not as much.
- No longer dispenses 3 sodas in sequence. Now, just one.
- On a bad result, fires a soda can at the worker at high velocity. This can kill.

Pile of Mail:
- Always give buffs on success, instead of stilling having a junk chance.
- Buffs have been improved significantly.
- Work mechanics have been changed slightly, repression work is now possible but ill-advised.
- Removed misleading wording on the abnormality's info page, and updated it to reflect new mechanics.
- Added new junk mail and ways to receive it.

Quiet Day:
- Applies buff to both the user and whoever may be sitting next to it.

Sleeping Beauty:
- Buff Duration does not tick down while you are asleep, and lasts longer if you are at least resting.
- Buff now heals every tick instead of 50% of the time every tick.
- Buff heal is massively increased if you're sleeping and slightly buffed if you are resting.

We Can Change Anything:
- Sacrifice now caps out at 2000 PE in case someone wants to keep spamming it for some reason.
- Sacrifice actually helps the facility meet the quota, but only 50%.
- Sacrifice now has a -500% work rate as opposed to 100. It turns out that those successful blips were adding PE in addition to the up to 1000 being generated by the work itself.

## Why It's Good For The Game
Better ZAYINs

## Changelog
:cl:
balance: rebalanced ZAYINs
/:cl:

### CODE REVIEW

File: code/modules/paperwork/records/info/zayin.dm [hunk 1]
```diff
@@ -71,7 +71,7 @@
 	abno_code = "D-01-145"
 	abno_info = list(
 		"When work with T-01-145 was completed, it started to speak.",
-		"If the agent working with T-01-145 listened to it's entire story, they received a buff for the corresponding attribute, lasting 5 minutes.")
+		"If the agent working with T-01-145 listened to it's entire story, everyone in the room received a buff for the corresponding attribute, lasting 5 minutes.")
```
Comment (Kitsunemitsu) [inline code/modules/paperwork/records/info/zayin.dm:74]: It's a 5 minute +15 stat buff we don't need to cram like 15 people into one room to give them all buffs.
I've said this much so far and already.

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]
This Quiet Day buff ain't gonna fly.

**Comment by InsightfulParasite** [comment]
Im going to put this on the month of abnormalities milestone since its technically abnormality focus'd

File: code/modules/mob/living/simple_animal/abnormality/zayin/oceanwaves.dm [hunk 2]
```diff
@@ -168,121 +162,123 @@
 		L.reagents?.add_reagent_list(current_reagents)
 	return ..()
 
-// Soda cans
+// Good Soders
```
Comment (Kitsunemitsu) [inline code/modules/mob/living/simple_animal/abnormality/zayin/oceanwaves.dm:165]: Love what you're doing, but the mystery is what makes Ocean waves mechanic actually work.
If you knew you were drinking something good every time you drank it, without a clerk processing it, then that makes Oceanic Waves significantly more powerful.

It's already picked as much as Wellcheers is

Comment (Coxswain-Navigator) [reply code/modules/mob/living/simple_animal/abnormality/zayin/oceanwaves.dm:165]: I will find a way to nerf it a bit. I don't like the current implementation of this abnormality.

File: code/modules/mob/living/simple_animal/abnormality/zayin/oceanwaves.dm [hunk 3]
```diff
@@ -168,121 +157,130 @@
...
-		/datum/reagent/toxin/mutetoxin = 10,
-	)
-
-//Random healing soders
-/obj/item/reagent_containers/food/drinks/soda_cans/oceanwave/oxan
-	list_reagents = list(
-		/datum/reagent/consumable/wellcheers_purple/oceanwave = 5,
-		/datum/reagent/medicine/oxandrolone = 10,
+		/datum/reagent/drug/methamphetamine = 10,
+		/datum/reagent/toxin/fentanyl = 5,
```
[the remark this answers is not in the document]
Comment (Coxswain-Navigator) [reply code/modules/mob/living/simple_animal/abnormality/zayin/oceanwaves.dm:233]: Good point. I'll add a chemical to one that treats brain damage.

**Review by Kitsunemitsu — APPROVED** [review]
Everything here looks good. I'll approve it.

**Pull request merged**